core-bundle/contao/classes/BackendUser.php

```diff
@@ -155,6 +155,8 @@ public function __get($strKey)
 	 */
 	public function hasAccess($field, $array)
 	{
+		trigger_deprecation('contao/core-bundle', '5.2', 'Using "%s()" has been deprecated and will no longer work in Contao 6. Use the "ContaoCorePermissions::USER_CAN_ACCESS_*" permissions instead.', __METHOD__);
+
 		if ($this->isAdmin)
```

Comment:
Should we forward those to the new voter? Or is it too complicated?

Reply:
I don't think so. We don't do that on the other methods, and technically you could use this method on a User object that is not the current Symfony security user.